src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -116,8 +121,7 @@ def update_alert_rule(
             # The user has requested a new Slack channel and we tell the client to check again in a bit
             return Response({"uuid": client.uuid}, status=202)
         else:
-            alert_rule = serializer.save()
-            return Response(serialize(alert_rule, request.user), status=status.HTTP_200_OK)
```

Comment:
small hack here: the serializer class is broken due to this change https://github.com/getsentry/sentry/pull/84753

that change introduced a broken diamond inheritance where the serializer has two disparate types -- I'm actually a little surprised mypy didn't complain about overriding the generic twice (and I imagine will in a future version).   this avoids the reassignment which avoids the "error" but the class itself is still unsound